CMakeLists.txt

```diff
@@ -467,8 +467,8 @@ set(system_test_runner_sources
   "source/tests/system/nidaqmx_session_tests.cpp"
   "source/tests/system/nidcpower_driver_api_tests.cpp"
   "source/tests/system/nidcpower_session_tests.cpp"
-  "source/tests/system/nidigital_driver_api_tests.cpp"
   "source/tests/system/nidigital_session_tests.cpp"
+  "source/tests/system/nidigital_driver_api_tests.cpp"
```

Comment:
Can you put this back above "source/tests/system/nidigital_session_tests.cpp" like it was before? That'll get rid of this file in the diff.